packages/shared/src/components/squads/utils.tsx

```diff
@@ -4,7 +4,7 @@ import { PromptOptions } from '../../hooks/usePrompt';
 import { SquadForm } from '../../graphql/squads';
 
 export enum ModalState {
-  Details = 'Squad details',
```

Comment:
Is this actually still used? Not quite sure we use this flow anymore.

Reply:
It is, but below it `modalStateToScreenValue`, left it because I did not know the context.

Reply:
Ah let's leave it might be needed for typesafety or something